Examples/MoveUserToAnotherAccount.cs

```diff
@@ -0,0 +1,126 @@
+        /// specified name and creates the new user inside it.</summary>
+
+        /// <param name="wrapper">The ApiWrapper object to use</param>
+        /// <param name="uername">The username of the user to move</param>
+        /// <param name="newAccountName">The name of the account to move the user across to, or create</param>
+        /// <param name="companyName">The company name of the new account</param>
+        /// <param name="countryCode">The country code of the new account</param>
+        /// <param name="email">The email address of the new user</param>
+        public static async Task DoWork(ApiWrapper wrapper,
+            string username,
+            string newAccountName,
+            string companyName,
+            string countryCode,
+            string email)
+        {
+            SubscriberDataModel subscriberModel = await GetSubscriber(wrapper,
+                username,
+                email,
+                companyName,
+                countryCode,
+                newAccountName);
+
+            await wrapper.PutSubscriberAsync(subscriberModel.AccountName,
+                username,
+                CreateSubscriberDataForm(subscriberModel, newAccountName));
+
+            Console.WriteLine($"Put '{username}' in '{newAccountName}' account");
+
+        }
+
+        /// <summary>Gets the subscriber object of the desired user. if the user doesnt exist,
+        /// it creates a new SubscriberDataModel object with the specified email address and account name.
+        /// it loops through all the accounts, and all the subscribers in each account to find the desired user</summary>
+
+        /// <param name="wrapper">The ApiWrapper object to use</param>
+        /// <param name="uername">The username of the user to move</param>
+        /// <param name="newAccountName">The name of the account to move the user across to, or create</param>
+        /// <param name="companyName">The company name of the new account</param>
+        /// <param name="countryCode">The country code of the new account</param>
+        /// <param name="email">The email address of the new user</param>
+        private static async Task<SubscriberDataModel> GetSubscriber(ApiWrapper wrapper,
+            string username,
+            string newAccountName,
+            string companyName,
+            string countryCode,
+            string email)
+        {
+            foreach (AccountSearchResults accountResult in await wrapper.GetAllAvailableAccountsAsync())
+            {
+                foreach (SubscriberDataModel subscriber in await wrapper.GetSubscribersAsync(accountResult.Name))
+                {
+                    if (subscriber.Username == username)
+                    {
+                        Console.WriteLine($"User '{username}' exists");
+                        return subscriber;
+                    }
+                }
+            }
+
+            await CreateAccount(wrapper, companyName, countryCode, newAccountName);
```

Comment:
Do we always want to create a new account if the user does not exist? What if the user does not exist but the desired account does?